recipes/cf_xarray/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "cf_xarray" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xarray-contrib/cf-xarray/archive/v{{ version }}.tar.gz
+  sha256: b6fa9d0f3a97306da2a2fb72b8774241c4a0a1e022363567b8f340b711df40a6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - python >=3.6
+    - setuptools
```

Comment:
I think this section isn't needed

Reply:
thanks! looks like it may succeed this time around.